boxxy/content/afterparty.html

```diff
@@ -1,12 +1,10 @@
-<div class="schedule row">
-    <div class="thirteen columns offset-by-five">
-        <table>
-          <tr><th>21u00</th><td>Eindceremonie met fakkel- en vlaggentocht,<br />begeleid door de Gentse Studentenfanfare</td></tr>
-          <tr><th>21u15</th><td>Bart Ricardo</td></tr>
-          <tr><th>22u45</th><td>Bekendmaking winnaars 12urenloop en prijsuitreiking</td></tr>
-          <tr><th>23u00</th><td>Winnaar DJ-contest</td></tr>
-          <tr><th>23u30</th><td>DJ Natick</td></tr>
-          <tr><th>01u00</th><td>Chillout mix by Fou Concept</td></tr>
-        </table>
-    </div>
+<div class="sixteen columns">
```

Comment:
Not really, `.row` only adds a a bot-margin of 20px and does something with nested colums (which we don't have). We could leave it there if we want a bit more bottom margin between the table and the site footer.